java/solrmarc_mixin/Makefile

```diff
@@ -9,7 +9,7 @@ SRC=src
 VUFIND_SRC=/usr/local/vufind/import/index_java/src
 PACKAGE_DIRECTORY=de/unituebingen/ub/ubtools/solrmarcMixin
 PACKAGE_DIRECTORY_VUFIND=org/vufind/index
-LIB_DIRECTORY=lib/*:/usr/local/vufind/solr/lib/*:/usr/local/vufind/solr/jetty/lib/ext/*:/usr/local/vufind/solr/jetty/work/jetty-0.0.0.0-8080-solr.war-_solr-any-/webapp/WEB-INF/lib/*:/usr/local/vufind/import/lib/*:/usr/local/vufind/import/lib_local/*:/usr/local/ub_tools/java/lib/*:/usr/local/vufind/solr/vendor/dist/solrj-lib/*
+LIB_DIRECTORY=lib/*:/usr/local/vufind/solr/lib/*:/usr/local/vufind/solr/jetty/lib/ext/*:/usr/local/vufind/solr/jetty/work/jetty-0.0.0.0-8080-solr.war-_solr-any-/webapp/WEB-INF/lib/*:/usr/local/vufind/import/lib/*:/usr/local/vufind/import/lib_local/*:/usr/local/ub_tools/java/lib/*:/usr/local/vufind/solr/vendor/dist/solrj-lib/*:/usr/local/vufind/solr/vendor/server/solr-webapp/webapp/WEB-INF/lib/lucene-core-7.3.1.jar
```

Comment:
Wouldn't it be advantageous to include the whole directory with '*' to avoid a dependency on a specific lucene version?